neqo-transport/src/fc.rs

```diff
@@ -255,6 +273,11 @@ where
         }
     }
 
+    const fn should_send_flowc_update(&self) -> bool {
```

Comment:
```suggestion
    const fn should_send_update(&self) -> bool {
```

The files is `fc.rs` and the type is `ReceiverFlowControl`

Reply:
Renamed in 200aaefe63bf5583585be85a0240880df0d3c5ad.